### PR TITLE
fix: Update Gemini version in all FC and RE notebooks; fix multimodal FC notebook

### PR DESCRIPTION
**Pull request by koverholt**
# Description

This PR:

* Updates model to `gemini-1.5-pro-002` in all Function Calling and Reasoning Engine notebooks
* Uncomments the first two cells that were inadvertently skipped in the multimodal_function_calling notebook

- [X] Follow the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/CONTRIBUTING.md).
- [X] You are listed as the author in your notebook or README file.
  - [X] Your account is listed in [`CODEOWNERS`](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/.github/CODEOWNERS) for the file(s).
- [X] Make your Pull Request title in the <https://www.conventionalcommits.org/> specification.
- [X] Ensure the tests and linter pass (Run `nox -s format` from the repository root to format).
- [X] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: gemini/function-calling/forced_function_calling.ipynb [hunk 1]
```diff
@@ -1,817 +1,830 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "id": "c8922eacdb2d"
+   },
+   "outputs": [],
+   "source": [
+    "model = GenerativeModel(\n",
+    "    \"gemini-1.5-pro-002\",\n",
```
Comment (holtskinner) [inline gemini/function-calling/forced_function_calling.ipynb:283]: Do any of these explicitly require a numbered version?

We've been trending towards using the auto-updating version for notebooks `gemini-1.5-pro`

**Review by holtskinner — APPROVED** [review]
See comment about versions

File: gemini/function-calling/forced_function_calling.ipynb (hunk 1 above)
Comment (koverholt) [reply gemini/function-calling/forced_function_calling.ipynb]: Thanks for the reminder about this, I noticed that across other notebooks. It comes from habit of dealing with early versions and flakiness in Function Calling and Reasoning Engine. But things have stabilized enough for now that I'll flip all of these to `gemini-1.5-pro`.

**Comment by koverholt** [comment]
@holtskinner, I'm running into spurious Pylint errors that pop up whenever I touch the SQL Talk app. Other than the spurious errors, all other tests are passing. Would you mind merging this when you have a chance? Thanks!

**Comment by holtskinner** [comment]
> @holtskinner, I'm running into spurious Pylint errors that pop up whenever I touch the SQL Talk app. Other than the spurious errors, all other tests are passing. Would you mind merging this when you have a chance? Thanks!

Don't worry about that, some of the criteria have been updated since it was published.

**Pull request merged**